products/antixlinux.md

```diff
@@ -0,0 +1,60 @@
+---
+permalink: /antix
+alternate_urls:
+-   /antixlinux
+-   /antix-linux
+title: antiX
+category: os
+releasePolicyLink: https://www.antixforum.com/forums/topic/when-is-end-of-support-for-stable-antix-versions-17-19/#post-26424
+iconSlug: NA
+activeSupportColumn: true
+versionCommand: cat /etc/os-release
+releaseColumn: true
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
+purls:
+-   purl: pkg:os/mxlinux
+auto:
+-   distrowatch: antix
+    regex: '^Distribution Release: antiX (?P<major>\d)\.(?P<minor>\d)$'
+releases:
+-   releaseCycle: "21"
+    codename: "Grup Yorum"
+    releaseDate: 2021-10-31
+    latest: "22"
+    latestRelaseDate: 2022-10-19
+    support: 2024-06-30
+    eol: 2026-06-30
+    link: https://antixlinux.com/antix-22-released/
+-   releaseCycle: "19"
+    codename: "Marielle Franco / Hannie Schaft / Manolis Glezos / Grup Yorum"
+    releaseDate: 2019-10-17
+    latest: "19.4"
+    latestRelaseDate: 2021-05-21
+    support: 2022-09-10
+    eol: 2024-06-30
+    link: https://mxlinux.org/blog/mx-19-4-now-available/
```

Comment:
wrong link, probably you mixed with mx linux

Reply:
@usta, good catch! I believe this has been corrected in the latest update to the [antix branch](https://github.com/alexkoster/endoflife.date/tree/antix) but please let me know if you'd like me to make any further changes.